### PR TITLE
created the onboarding view component

### LINKED ISSUE #38: Create the onboarding view component

### PR DESCRIPTION
**Pull request by 7malikk**
### What type of PR is this? (check all applicable)

- [ ] Feature
- [ ] Refactor
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [x] Other: Component
  
  
### Description of what this Pull Request does
- Created the onboarding view component

### Related Tickets & Documents
- Fixes #38 
  

### Added/updated tests?
- [ ] Yes
- [ ] Not Yet, and here is why: __explain here__
- [ ] I need help with tests
- [x] No, and here is why: No testing at this point

### CODE REVIEW

File: src/components/onboarding_comp.js
Comment (annonymousauthority) [inline src/components/onboarding_comp.js:1]: Not comfortable with custom color codes for text color, please revert to the instructions given earlier

File: src/components/onboarding_comp.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        Get Started With Carai
+      </div>
+      <ul className="marker:text-[#6D6AEC] list-disc px-6 space-y-6 font-bold text-lg">
+        <li>Share, Review & Improve your Code.</li>
+        <li>Connect with peers and code together.</li>
+        <li>Communicate with peers and code together.</li>
+        <li>Keep track of changes with code versionong.</li>
+        <li>Stop coding alone and start collaborating today.</li>
+      </ul>
+      <div className="flex flex-col space-y-6 ">
```
Comment (annonymousauthority) [inline src/components/onboarding_comp.js:14]: This should be `space-y-3` not `space-y-6`

File: src/components/onboarding_comp.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+function Onboarding() {
+  return (
+    <div className="space-y-6">
+      <div className="text-[#6D6AEC] text-[43px] font-bold">
+        Get Started With Carai
+      </div>
+      <ul className="marker:text-[#6D6AEC] list-disc px-6 space-y-6 font-bold text-lg">
```
Comment (annonymousauthority) [inline src/components/onboarding_comp.js:7]: There's a font-size and font-weight difference please note that in this view

**Review by annonymousauthority — CHANGES_REQUESTED** [review]

File: src/components/onboarding_comp.js
Comment (7malikk) [reply src/components/onboarding_comp.js:1]: The custom colors are from the figma, but I'd revert them to the `blue-500`

Could the `hover:` be `blue-600`?

File: src/components/onboarding_comp.js (hunk 1 above)
Comment (7malikk) [reply src/components/onboarding_comp.js:14]: Noted

File: src/components/onboarding_comp.js
Comment (annonymousauthority) [reply src/components/onboarding_comp.js:1]: yes

**Review by annonymousauthority — CHANGES_REQUESTED** [review]

File: src/components/onboarding_comp.js (hunk 2 above)
Comment (7malikk) [reply src/components/onboarding_comp.js:7]: These were based on the Figma design sizes, should I revert?

**Review by annonymousauthority — APPROVED** [review]
I'd point out you take not, not all text in the list are same font-size and font-weight
So you'll be asked to revisit soon.

**Pull request merged**